test/integration/rule-content-route.yaml.tmpl

```diff
@@ -1,6 +1,9 @@
-destination: "{{.destination}}.{{.namespace}}.svc.cluster.local"
+destination: {{.destination}}.{{.namespace}}.svc.cluster.local
 precedence: 2
 match:
+  source: {{.source}}.{{.namespace}}.svc.cluster.local
+  source_tags:
+    version: v1
   http:
```

Comment:
No. Two versions of hello, hello v1 can only call world v1, so on. Otherwise this test as is, is not checking whether service hello-v2 can access service world-v1 or not.